spec/requests/api/v1/api_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe RedeemifyCodesController, type: :request do
+  describe "GET /users/:user_id/redeemify_codes/:code" do
```

Comment:
looks find, but we could move the get into a subject and then create separate `it` statements for each assertion